src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
Ok, I have to admit this doesn't make any sense to me.
So right now, I have
```
@property(nonatomic, readonly, nonnull) id<MDMTransitionContext> transitionContext;
```
to save the instance of ```MDMTransitionContext```. If I inherit it, all I am doing is adding wrappers around ```MDMTransitionContext``` and I still have to save the instance so I can implement ```transitionDidEnd```.